site/templates/features.php

```diff
@@ -21,23 +21,23 @@
           ]) ?>
 
           <?= snippet('features/text', [
-            'heading' => 'Make it your own',
+            'heading' => 'Make it yours',
             'text' => '
-              Kirby let\'s you create a control panel for yourself and your editors that is tailored to your site. Make the Panel adapt to your users, content and use cases - not the other way around.
+              Kirby lets you create a control panel for yourself and your editors that is tailored to your site. Make the Panel adapt to your users, your content and your use cases—not the other way around.
             '
           ]) ?>
 
           <?= snippet('features/text', [
             'heading' => 'Drafts',
             'text' => '
-              Prepare your content - pages, articles, events, products - and only publish them when everything is in place with the click of a button. Send preview links to others so they can review your drafts even before they go live.
+              Prepare your content—pages, articles, events, products—and only publish them when everything is in place with the click of a button. Send preview links to others so they can review your drafts before they go live.
             '
           ]) ?>
 
           <?= snippet('features/text', [
             'heading' => 'Drag & Drop sorting',
             'text' => '
-              Sorting shouldn\'t be complicated. Just pick your pages or files up and drop them where you want them to be. Why should it be anymore complicated than that?
+              Sorting pages or files is a breeze. Just pick them up and drop them where you want them to be. It shouldn\'t be more complicated than that?
```

Comment:
Keeping the question mark at the end intended?

Reply:
I actually think the whole question is redundant and should be replaced with something better. I didn't have a good idea yet.